### DISCUSSION TITLE
【LINE】Googleドライブの画像をLineに送りたい（ImageSendMessage）

### DISCUSSION THREAD

**Discussion by SuzumitsuTs1103** [1ft-seabass/ma-hackathon-qa-and-knowledge-202210#22] [Q&A] [answered]
Googleドライブを共用設定にして、その画像をLineに送りたい（ImageSendMessage）のですが
潰れて、表示できません。
容量も100KBくらいです。
⇨チームで撮った素材を表示させたいだけです。
（素材屋にある画像は表示できるので、コーディングは正しそう）

@1ft-seabass

**Comment by 1ft-seabass** [comment, marked as answer]
お！少ししたらお答えしますね。ざっくりいうと、Google ドライブ内の画像を読むのは大変なので、Gyazoというサービスを使う方法をアドレスしますー。

**Reply by 1ft-seabass** [reply]
@SuzumitsuTs1103 ↑

**Reply by 1ft-seabass** [reply]
まず無料で登録できるので Gyazo に登録お願いします～。

アカウントの作成
https://helpfeel.com/GyazoJP/%E3%82%A2%E3%82%AB%E3%82%A6%E3%83%B3%E3%83%88%E3%81%AE%E4%BD%9C%E6%88%90-5d3807bdd8b95000171654e0

**Reply by 1ft-seabass** [reply]
その上で、お手数ですが、チームの素材をためしに2～3枚アップロードしてみてください。
https://helpfeel.com/GyazoJP/%E7%94%BB%E5%83%8F%E3%83%95%E3%82%A1%E3%82%A4%E3%83%AB%E3%81%AE%E3%82%A2%E3%83%83%E3%83%97%E3%83%AD%E3%83%BC%E3%83%89-5d3807bfd8b9500017165507

**Reply by 1ft-seabass** [reply]
各画面の詳細でアップロードした画像の直URLが取得できるので、それを LINE BOT の画像オブジェクトに反映すれば見えるはずです～。

**Reply by 1ft-seabass** [reply]
（直URLの取得が UI 的に分かりにくいかもなので、必要であれば、説明しまっす！） @SuzumitsuTs1103

**Comment by SuzumitsuTs1103** [comment]
できました。ありがとうございます！！！！！

**Reply by 1ft-seabass** [reply]
おおお！最高！よかったですー。
